tests/installation/partitioning_crypt_no_lvm.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test installation with encrypted drive, but without lvm
+# This is possible only with storage-ng. See poo#26810
+#
+# Maintainer: Rodion Iafarov <riafarov@suse.com>
+
+use strict;
+use base "opensusebasetest";
+use testapi;
+use version_utils 'is_storage_ng';
+use partition_setup 'enable_encryption_guided_setup';
+
+sub run {
+    # Encrypting without lvm is only possible with storage_ng
+    return unless is_storage_ng;
```

Comment:
better die here with the message you provided. If this module is scheduled for a product which does not support it I guess there is no use in scheduling it and we want to know that failure, right?

->
```perl
die "Encrypting without lvm is only possible with storage_ng" unless is_storage_ng;
```

Reply:
Makes sense, thanks!